src/common/transformations/include/transformations/low_precision/mark_dequantization_subgraph.hpp

```diff
@@ -76,5 +76,19 @@ class TRANSFORMATIONS_API KeepConstPrecision : public ov::pass::MatcherPass {
                                 bool fold_multiply_const = true);
 };
 
+/**
+ * @ingroup ov_transformation_common_api
+ *
+ * @brief KeepDequantizationPrecision matches Dequantization subgraphs and, if precision matches with
+ * specified, Convert, Multiply (and Subtract) nodes might be marked with disable_fp16_compression attribute.
+ * This prevents precision loss when the original precision is lowered during ConstantFolding execution.
+ *
```

Comment:
Can we add something like that? please check before insertion

Example scenario:
original graph:
 i32 → Convert(f32) → Subtract(zero_point_f32) → Multiply(scale_f32)
without marking (KeepDequantizationPrecision), ConvertPrecision transformation may convert the weights from fp16 to fp32, it will lead to precision lost:
 i32 → Convert(f16) → Subtract(zero_point_f16) → Multiply(scale_f16)

Reply:
Yeah, I have added the example scenario to the description